flutter_mobx/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.0+2
```

Comment:
I think its a patch release ... so make it `2.1.1` please

Reply:
Also pls run set_versions again

Reply:
@pavanpodila done